frontend/homework/8/index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bonus Goal</title>
+    <link rel="stylesheet" href="style.css">
+    <script src ="code.js"></script>
+</head>
+<body>
+    <div class = "title-main">
+       <h2> Title</h2>
+    </div>
+    <div class = "search-box-container">
+        <input type="text" placeholder="Find a recipe.." class = "input-box">
+        <button id = "search-btn">Search</button>
+    </div>
+    <div class ="results">
+
+    </div>
+    <script>
```

Comment:
Avoid inline scripts.
Set `allowJs=true` in config and manipulate accordingly for TS-JS inter-communication